packages/DataGrid/src/components/WhenScrollBarReachedBottom/WhenScrollBarReachedBottom.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function WhenScrollBarReachedBottom(props) {
```

Comment:
I'd love for someone with better English than me to come up with better names for these two components. `Basement`? 😄 Just `Bottom`? `Footer`?
Then `End` would be e.g. `FooterWrapper`.

Reply:
I ended using Basement, I think is flexible enough. and oscar was 👍 as well...